nav-app/src/app/tabs/tabs.component.html

```diff
@@ -48,17 +48,60 @@
     </div> -->
     <div class="newTab border">
         <mat-accordion class="headers-align">
-            <mat-expansion-panel #newlayer hideToggle="true" (click)="newLayer(); newlayer.close()">
+            <mat-expansion-panel #newlayer>
                 <mat-expansion-panel-header>
                     <mat-panel-title>
                         Create New Layer
                     </mat-panel-title>
                     <mat-panel-description>
                         Create a new empty layer
                     </mat-panel-description>
-
                 </mat-expansion-panel-header>
-                <!-- this content will never be shown because clicking opens a new layer tab -->
+
+                <table class="inputTable">                        
+                    <ng-template ngFor let-domain [ngForOf]="filterDomains(ds.versions[0])" let-domainIndex="index">
+                        <td><button mat-button (click)="newLayer(domain.id)">{{domain.name}}</button></td>
+                    </ng-template>
+                </table>
+
+                <mat-expansion-panel>
+                        <mat-expansion-panel-header>
+                            <mat-panel-title>
+                                More Options
+                            </mat-panel-title>
+                        </mat-expansion-panel-header>
+
+                    <table class="layerOpTable">
+                        <tr>
+                            <td>
+                                <mat-form-field>
+                                    <mat-select placeholder="version" [(ngModel)]="nVersion" required>
+                                        <mat-option [value]="null">none</mat-option>
+                                        <mat-option *ngFor="let nVersion of this.dataService.versions" [value]="nVersion">{{nVersion}}</mat-option>
+                                    </mat-select>
+                                </mat-form-field>
+                            </td>
+                            <td>
+                                Choose the version for the new layer. <em>*Versions prior to {{ds.versions[ds.versions.length-1]}} are not supported by Navigator v{{nav_version}}.</em>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <mat-form-field>
+                                    <mat-select placeholder="domain" [(ngModel)]="nDomain" [disabled]="!nVersion" required>
+                                        <mat-option [value]="null">none</mat-option>
+                                        <mat-option *ngFor="let nDomain of filterDomains(nVersion)" [value]=nDomain>{{nDomain.name}}</mat-option>
+                                    </mat-select>
+                                </mat-form-field>
```

Comment:
I love this! The dropdowns are much more intuitive than the mock-ups I had created.